api/src/main/java/org/pmiops/workbench/exfiltration/impl/EgressSumologicRemediationService.java

```diff
@@ -72,9 +72,15 @@ public EgressSumologicRemediationService(
   }
 
   @Override
-  protected void sendEgressRemediationEmail(DbUser user, EgressRemediationAction action)
-      throws MessagingException {
-    mailService.sendEgressRemediationEmail(user, action);
+  protected void sendEgressRemediationEmail(
+      DbUser user, EgressRemediationAction action, DbEgressEvent event) throws MessagingException {
+    SumologicEgressEvent originalEvent = egressEventMapper.toSumoLogicEvent(event);
+    String serviceName =
```

Comment:
maybe a bit more optimized and readable version would look like...not tested though.

```
String srcGkeServiceName = originalEvent.getSrcGkeServiceName();
    Optional<AppType> appTypeOpt = StringUtils.isNotEmpty(srcGkeServiceName) ? appServiceNameToAppType(srcGkeServiceName) : Optional.empty();
    String serviceName = appTypeOpt.isPresent() ? appTypeOpt.get().toString() : "Jupyter";
```

Reply:
ah nice. update a bit and make it readable